### ISSUE TITLE
Update-Database with specific connection string

### ISSUE THREAD

**Issue by alemag1986** [aspnet/EntityFramework.Docs#857]
I can't find a way of updating my database in all the different environments that my app runs. 
Can you please provide some documentation in how to achieve this?
I've several appsettings and I need to be able to run Update-Database on each environment. Thx!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6e28b0a8-b7e4-4c07-9e60-0df403bbee28
* Version Independent ID: 12300b31-6f7b-b75c-970d-693365e77013
* Content: [Migrations - EF Core](https://docs.microsoft.com/en-us/ef/core/managing-schemas/migrations/#feedback)
* Content Source: [entity-framework/core/managing-schemas/migrations/index.md](https://github.com/aspnet/EntityFramework.Docs/blob/master/entity-framework/core/managing-schemas/migrations/index.md)
* Product: **entity-framework**
* GitHub Login: @bricelam
* Microsoft Alias: **bricelam**

**Comment by kduenke** [comment]
@alemag1986 It works much like web config transforms worked in the past.  Maybe this will help you understand how this works...
https://stackoverflow.com/questions/48913968/net-core-2-entity-framework-update-database-of-different-environment

In the same vein, it'd be nice to be able to drive settings through launchSettings.json as well.  Is this possible today?

**Comment by bricelam** [comment]
Blocked on dotnet/cli#9730

**Comment by alemag1986** [comment]
@kduenke thanks for the answer, yup I end up changing the NuGet Environment:
`$env:ASPNETCORE_ENVIRONMENT='my-environment'`

**Comment by ajcvickers** [comment]
Duplicate of #1359
